### PR TITLE
Enhance sfpshow to support RJ45 port

### PR DESCRIPTION
**Pull request by keboliu**
Signed-off-by: Kebo Liu <kebol@nvidia.com>

### CODE REVIEW

**Comment by stephenxs** [comment]
Do we need to add test case

File: scripts/sfpshow [hunk 1]
```diff
@@ -390,16 +397,19 @@ class SFPShow(object):
...
-
-        if dump_dom:
-            sfp_type = sfp_info_dict['type']
-            dom_info_dict = state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_DOM_SENSOR|{}'.format(interface_name))
-            dom_output = self.convert_dom_to_output_string(sfp_type, dom_info_dict)
-            output += dom_output
+        if self.is_rj45_port(interface_name):
+            output = interface_name + ': ' + 'SFP EEPROM is not applicable for RJ45 port' + '\n'
+        else:
+            sfp_info_dict = state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(interface_name))
```
Comment (Junchao-Mellanox) [inline scripts/sfpshow]: How about we do it like this:

```python
sfp_info_dict = state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(interface_name))
if sfp_info_dict ['type'] == RJ45_PORT_TYPE:
    output = interface_name + ': ' + 'SFP EEPROM is not applicable for RJ45 port' + '\n'
else:
    output = interface_name + ': ' + 'SFP EEPROM detected' + '\n'
    ...
```

**Comment by keboliu** [comment]
> Do we need to add test case

will add

File: scripts/sfpshow (hunk 1 above)
Comment (keboliu) [reply scripts/sfpshow]: fixed

**Pull request closed without merging**